superset/views/core.py

```diff
@@ -1182,6 +1182,11 @@ def explore(self, datasource_type, datasource_id):
                 "You don't have the rights to alter this slice",
                 status=400)
 
+        if action == 'saveas' and not slice_add_perm:
+            return json_error_response(
+                "You don't have the rights to create this slice",
```

Comment:
You be great to make this translatable using `_("Translatable")`. Also we're moving away from `slice` in favor of `chart`.
To minimize the number of strings to translate you may want to break it into chunks:
_("You don't have the rights to ") + _("create this ") + _("slice")